### ISSUE TITLE
Earthsblood Causes Neural Reimaging to Endlessly Consume Volume

### ISSUE THREAD

**Issue by IronEleven** [citadel-station/citadel-main#10194]
## Round ID:
20549

## Testmerges:
#9928: '[READY]Even more organ tweaks!' by Thalpy at commit 220cca7e76
#10119: '[TESTMERGE READY] Makes jukeboxes stream the songs instead of forcing instant downloads.' by Useroth at commit e3387eb389
#9596: 'Bring abductors up-to-date with /tg/.' by Linzolle at commit 1d3fdad459
#10011: 'Bloodsuckers, or, yknow, vampires.' by Arturlang at commit 6124fbf316
#10120: 'Night vision trait but it just boosts barely-lit visibility' by kevinz000 at commit c0f8a42e0f
#9751: 'RISING BASS, THE NEW MARTIAL ARTS' by Seris02 at commit dfc8a1dc73
#10177: 'NUCLEAR FIST. A new wizard spell involving punching people so hard they explode.' by BlackMajor at commit 59d0376985

## Reproduction:
Get augmented with nanites including Neural Reimaging, take a bite of ambrosia gaia.  Neural Reimaging will continuously consume nanite volume until removed.

**Comment by Putnam3145** [comment]
It's *probably* because earthsblood causes brain damage. Does it keep happening even after the effects have worn off?

**Comment by IronEleven** [comment]
Yeah seems to keep going well after the chemical's cleared your system, judging by how the colors and shaking were gone and nanites kept draining.
